examples/gatsby/src/html.js

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { createMediaStyle } from "./Media"
+
+export default function HTML(props) {
+  return (
+    <html {...props.htmlAttributes}>
+      <head>
+        <meta charSet="utf-8" />
+        <meta httpEquiv="x-ua-compatible" content="ie=edge" />
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1, shrink-to-fit=no"
+        />
+
+        {/* Inject @artsy/fresnel styles in to the head */}
+        <style>{createMediaStyle()}</style>
```

Comment:
As mentioned before, let’s always do this in a constant at the module scope level.